src/App.jsx

```diff
@@ -14,70 +16,69 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [good, setGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  const clearGood = () => {
+    setGood('');
+  };
 
-      <button
-        data-cy="ClearButton"
-        type="button"
-        className="delete ml-3"
-      />
-    </h1>
+  const renderItem = itemName => (
+    <tr
+      data-cy="Good"
+      className={classNames(
+        { 'has-background-success-light': (good === itemName) },
+      )}
+    >
+      <td>
+        <button
+          data-cy="RemoveButton"
+          type="button"
+          className={classNames(
+            'button',
+            { 'is-info': (good === itemName) },
+          )}
+          onClick={() => setGood(itemName)}
+        >
+          {good === itemName ? '-' : '+'}
+        </button>
+      </td>
 
-    <table className="table">
-      <tbody>
-        <tr data-cy="Good">
-          <td>
-            <button
-              data-cy="AddButton"
-              type="button"
-              className="button"
-            >
-              +
-            </button>
-          </td>
-
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Dumplings
-          </td>
-        </tr>
-
-        <tr data-cy="Good" className="has-background-success-light">
-          <td>
-            <button
-              data-cy="RemoveButton"
-              type="button"
-              className="button is-info"
-            >
-              -
-            </button>
-          </td>
+      <td data-cy="GoodTitle" className="is-vcentered">
+        {itemName}
+      </td>
+    </tr>
+  );
 
-          <td data-cy="GoodTitle" className="is-vcentered">
-            Jam
-          </td>
-        </tr>
+  return (
+    <main className="section container">
+      {good.length === 0
+        && (
+          <h1 className="title is-flex is-align-items-center">
+            No goods selected
+          </h1>
+        )}
 
-        <tr data-cy="Good">
-          <td>
+      {good.length > 0
```

Comment:
You can use ternary here

Reply:
@nazarmatsevych thank You for the thorough comments, despite late hour 😉